Moodis/Moodis/Feature/Login/LoginForm.cs

```diff
@@ -26,22 +34,22 @@ private void LoginButton_Click(object sender, EventArgs e)
         {
             if (string.IsNullOrWhiteSpace(usernameField.Text))
             {
-                errorLabel.Text = "Username field is empty!";
+                errorLabel.Text = usernameEmpty;
             }else if (string.IsNullOrWhiteSpace(passwordField.Text))
             {
-                errorLabel.Text = "Password field is empty!";
+                errorLabel.Text = passwordEmpty;
             }
             else
             {
                 if (loginViewModel.Authenticate(usernameField.Text, passwordField.Text) != null)
                 {
-                    CameraForm cameraWindow = new CameraForm();
+                    var cameraWindow = new CameraForm();
                     cameraWindow.Show();
                     this.Hide();
```

Comment:
Shouldn't the window be closed instead of being hidden ?

Reply:
Close function closes whole application